Resources/PO/SimpleForm/SimpleFormDemo.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Library    SeleniumLibrary
+Resource    ../../../Data/InputData.robot
+
+*** Variables ***
+${SIMPLE_FORM_DEMO_LINK} =    css=ul[class*='nav'] li > a[href='./basic-first-form-demo.html']
+${PANEL_HEADING} =    xpath=//div[contains(text(), 'Single')]
+${DROPDOWN_MENU} =    xpath=//*[@id="navbar-brand-centered"]/ul[1]/li[1]/ul
+${USER_MESSAGE_TEXT_AREA} =    css=#get-input #user-message
+${SHOW_MESSAGE_BUTTON} =    xpath=//*[@id="get-input"]/button
+${SUM_A} =    css=input[id=sum1]
+${SUM_B} =    css=input[id=sum2]
+${GET_TOTAL_BUTTON} =    xpath=//button[contains(text(), 'Get Total')]
+${TOTAL_LABEL} =   xpath=//label[contains(text(), 'Total')]
+
+*** Keywords ***
+Go to "Simple Form Demo" Page
```

Comment:
this method belongs to LandingPage --> because it is interacting with elements from there